### PR TITLE
Story portfolio v1

### PR DESCRIPTION
**Pull request by markandmarkup**
Note - currently missing Contact section for early review
Contact section now merged

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 .DS_Store
+portfolio-text-copy.txt
```
Comment (charliecog) [inline .gitignore:2]: if this wont ever be in the project best to remove it and host it somewhere else in your laptop

File: index.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta name="viewport" content="width=device-width" />
+	<link href="https://fonts.googleapis.com/css?family=Gelasio:400,500,600&display=swap" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css?family=Poppins:400,500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="normalize.css" />
```
Comment (charliecog) [inline index.html]: indentation slightly off here

File: index.html [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+<body>
+	<header>
+		<div class="header-top">
+			<img src="./images/mamu-logo2.png" alt="Mark and Markup Logo" class="logo" />
+			<h1>Mark<span class="accent-color">&amp;</span>Markup</h1>
+			<img src="./images/hamburger-icon.png" alt="Menu Icon" id="burger-menu"/>
+		</div>
+		
+		<nav>
+
```
Comment (charliecog) [inline index.html:20]: remove line break

File: index.html [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+			<h3>Full stack developer portfolio</h3>
+		</div>
+		<a href="#" class="a-box">View this project ></a>
+	</section>
+	
+	<section class="about">
+		<a href="#" id="about" class="a-section-anchor">&nbsp;</a>	
+		<div class="center-container">
+			<a href="#about" class="a-section-title"><h2>About</h2></a>
+			<div class="about-content">
```
Comment (charliecog) [inline index.html:43]: could probably do with one less layer of div and put the center-container class on the div with about-content

File: index.html [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+			</div>
+		</div>
+	</section>
+	
+	<section class="portfolio">
+		<a href="#" id="portfolio" class="a-section-anchor">&nbsp;</a>
+		<div class="center-container">
+			<a href="#portfolio" class="a-section-title"><h2>Project Portfolio</h2></a>
+			<p class="touchscreen-info">Tap an image for a project summary</p>
+			<div class="portfolio-content">
```
Comment (charliecog) [inline index.html:57]: same again with center-container - could probably strip out one extra layer of div

File: index.html [hunk 6]
```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
```
Comment (charliecog) [inline index.html]: add a lang attribute

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,520 @@
...
+body {
+	font-family: 'Gelasio', serif;
+	color: #0E1116;
+	background-color: #374B68;
+}
+
+header {
+	background-color: #374B68;
+	padding-top: 16px;
+	height: 160px;
```
Comment (charliecog) [inline styles.css]: check this is doing something, if not remove it

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,520 @@
...
+	width: 100%;
+	position: fixed;
+	top: 0;
+	z-index: 1;
+}
+
+.header-top {
+	text-align: center;
+	overflow: hidden;
```
Comment (charliecog) [inline styles.css]: check this is doing something, if not remove it

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,520 @@
...
+}
+
+.logo {
+	display: inline-block;
```
Comment (charliecog) [inline styles.css]: check this is doing something, if not remove it

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,520 @@
...
+	width: 800px;
+	margin: 0 auto;
+	padding: 0 30px;
+	font-size: 20px;
+	font-family: 'Poppins', sans-serif;
+	text-transform: uppercase;
+}
+
+nav ul {
+	list-style-type: none;
```
Comment (charliecog) [inline styles.css]: check this is doing something, if not remove it

File: styles.css [hunk 11]
```diff
@@ -0,0 +1,520 @@
...
+	margin: 0;
+	padding: 0;
+	overflow: hidden;
```
Comment (charliecog) [inline styles.css:112]: check this is doing something, if not remove it

File: styles.css [hunk 12]
```diff
@@ -0,0 +1,520 @@
...
+	line-height: 23px;
```
Comment (charliecog) [inline styles.css]: check this is doing something, if not remove it

File: styles.css [hunk 13]
```diff
@@ -0,0 +1,520 @@
...
+	background-image: url('./images/project-1-hero.jpg');
+    background-position: center;
+    background-repeat: no-repeat;
+	background-size: cover;
+	text-align: center;
+	margin-top: 160px;
+}
+
+.hero-caption {
+	display: inline-block;
```
Comment (charliecog) [inline styles.css]: check this is doing something, if not remove it

File: styles.css [hunk 14]
```diff
@@ -0,0 +1,520 @@
...
+	text-align: center;
```
Comment (charliecog) [inline styles.css]: check this is doing something, if not remove it

File: styles.css [hunk 15]
```diff
@@ -0,0 +1,520 @@
...
+	margin: 0;
+	padding: 20px 0;
+	height: 80px;
+	text-transform: uppercase;
+	font-weight: 400;
+	color: #C0D7E0;
+}
+
+h3 {
+	display: block;
```
Comment (charliecog) [inline styles.css]: h3 are already block by default

File: styles.css [hunk 16]
```diff
@@ -0,0 +1,520 @@
...
+
+section.about {
+	background-color: #374B68;
+	min-height: 600px;
+}
+
+.center-container {
+	width: 880px;
+	margin: 0 auto;
+	height: 100%;
```
Comment (charliecog) [inline styles.css:186]: check this is doing something, if not remove it

File: styles.css [hunk 17]
```diff
@@ -0,0 +1,520 @@
...
+}
+
+.about-content {
+	overflow: auto;
+	min-height: 470px;
```
Comment (charliecog) [inline styles.css:190]: check this is doing something, if not remove it

File: styles.css [hunk 18]
```diff
@@ -0,0 +1,520 @@
...
+}
+
+.about-content p {
+	width: 50%;
+	float: left;
+	padding: 0 30px;
+}
+
+.headshot {
+	display: block;
```
Comment (charliecog) [inline styles.css]: check this is doing something, if not remove it

File: styles.css [hunk 19]
```diff
@@ -0,0 +1,520 @@
...
+	font-size: 20px;
+	word-spacing: 2px;
+}
+
+section.portfolio {
+	background-color: #20385E;
+	min-height: 570px;
+}
+
+.touchscreen-info {
```
Comment (charliecog) [inline styles.css:217]: is this still being used?

File: index.html [hunk 20]
```diff
@@ -0,0 +1,118 @@
...
+				<p>Take a look at some of my projects below to see what I can do, and feel free to get in touch if you'd like to work together! Bonus points if you're a fan of good food, squash (the sport), or scuba diving.</p>
+			</div>
+		</div>
+	</section>
+	
+	<section class="portfolio">
+		<a href="#" id="portfolio" class="a-section-anchor">&nbsp;</a>
+		<div class="center-container">
+			<a href="#portfolio" class="a-section-title"><h2>Project Portfolio</h2></a>
+			<p class="touchscreen-info">Tap an image for a project summary</p>
```
Comment (charliecog) [inline index.html:56]: is this still being used?

File: styles.css [hunk 21]
```diff
@@ -0,0 +1,520 @@
...
+.project5 {
+	background-image: url('./images/project-5-sml.jpg');
+}
+
+.project6 {
+	background-image: url('./images/project-6-sml.jpg');
+}
+
+.project-caption {
+	width: 100%;
```
Comment (charliecog) [inline styles.css]: only need 1 out of the width and height so remove the other

**Review by charliecog — COMMENTED** [review]
Looks good so far - almost there - comments inline

File: styles.css (hunk 11 above)
Comment (markandmarkup) [reply styles.css:112]: Overflow-hidden is stopping the scroll bar white box from appearing

File: styles.css (hunk 16 above)
Comment (markandmarkup) [reply styles.css:186]: Yes - it stretches out the portfolio section content to fill the whole section height

File: styles.css (hunk 19 above)
Comment (markandmarkup) [reply styles.css:217]: The touchscreen info p? It appears as a UI hint in tablet mode only. Hence display:none here.

File: index.html (hunk 20 above)
Comment (markandmarkup) [reply index.html:56]: Touchscreen info appears in tablet view only.

File: styles.css (hunk 21 above)
Comment (markandmarkup) [reply styles.css]: Removing height makes the dark background disappear, but width can be removed.

**Pull request merged**